Documentation.md

```diff
@@ -140,7 +140,14 @@ everything set up correctly and that you are able to listen for events.
         // initialize the ccp
         connect.core.initCCP(containerDiv, {
           ccpUrl: instanceURL,            // REQUIRED
-          loginPopup: true,               // optional, defaults to `true`
```

Comment:
Could you please also update the `README.md` file as well? Also, please run `npm run release` to update the build files that are in our `release` folder. We will automate this in the future but for now it's a good idea to run this every time a change is made so that they're not out of sync.

Reply:
Updated `README.md` to match the changes in `Documentation.md` and rebuilt the release files.